samples/azure-function/emissions-azure-function/function.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <AzureFunctionsVersion>v3</AzureFunctionsVersion>
```

Comment:
Shouldn't be this `<AzureFunctionsVersion>v4</AzureFunctionsVersion>`?

Reply:
I updated